src/tokens/universal.variables.json

```diff
@@ -2783,14 +2787,6 @@
           "value": "#434fcf",
           "referencedVariable": "$schemes.primary"
         },
-        "hover-monochrome": {
-          "type": "color",
-          "value": "rgba(0, 0, 0, 0.1000)"
-        },
-        "hover-primary": {
-          "type": "color",
-          "value": "rgba(69, 81, 209, 0.1000)"
-        },
```

Comment:
Is this not being used anywhere?

Reply:
No, it's a color token I had added just for design and use in Figma mocks for buttons, but I solved it in a better way without needing these